docs/dev/guidelines/client.rst

```diff
@@ -63,7 +63,14 @@ Some general aspects:
 1. Use single quotes for strings.
 2. All strings visible to the user need to be localized (make an entry in the corresponding ``*.json`` file).
 
-8. Style
+8. Buttons and Links
+========
+
+1. Be aware that Buttons navigate only in the same tab while Links provide the option to use the context menu or a middle-click to open the page in a new tab. Therefore:
+2. Buttons are best used to trigger certain functionalities
+3. Links are best for navigating on Artemis
+
```

Comment:
I think this is great but if you look at the other Documentation points they include examples. So maybe show an example like:
Dont do: <button (click)="openSomeOtherPage()"> ... <\/button>
Do: <a [routerLink]="getLinkToSomeOtherPage()"
            [queryParams]="getQueryParamsForPage()"> ... <\/a>

Reply:
done